packages/notifier/src/publish-kit.js

```diff
@@ -4,10 +4,19 @@
 import { HandledPromise, E } from '@endo/eventual-send';
 import { makePromiseKit } from '@endo/promise-kit';
 import { Far } from '@endo/marshal';
+import { M } from '@agoric/store';
+import { canBeDurable, vivifyFarClassKit } from '@agoric/vat-data';
 
 import './types.js';
 
+const { details: X, quote: q } = assert;
+
 const sink = () => {};
+const makeQuietRejection = reason => {
+  const rejection = HandledPromise.reject(reason);
+  rejection.catch(sink);
+  return rejection;
+};
 
 /**
  * Asyncronously iterates over the contents of a PublicationList as it appears.
```

Comment:
On line 24 below

>  * Thus, if no one else is holding on to those, it can be garbage collected.

Please clean up my singular vs plural confusion.